src/components/DynamicRacingBar/DynamicRacingBar.tsx

```diff
@@ -0,0 +1,154 @@
+import { useState, useEffect, useRef } from 'react';
+import * as echarts from 'echarts';
+import * as React from 'react';
+import PlayButton from './DynamicRacingBarin/PlayButton';
+import { getGithubTheme } from '../../utils/utils';
+const LIGHT_THEME = {
+  FG_COLOR: '#24292f',
+  BG_COLOR: '#ffffff',
+  PALLET: ['#5470c6', '#91cc75'],
+};
+
+const DARK_THEME = {
+  FG_COLOR: '#c9d1d9',
+  BG_COLOR: '#0d1118',
+  PALLET: ['#58a6ff', '#3fb950'],
+};
+const githubTheme = getGithubTheme();
+
+interface DynamicBarProps {
+  height: number;
+  legend1: string;
+  legend2: string;
+  yName1: string;
+  yName2: string;
+  data1: [string, number][];
+  data2: [string, number][];
+}
+const [playing, setPlaying] = useState(false);
```

Comment:
Hi @smitsekhadiaa, you cannot put `useState()` outside of a react functional component.

This causes the error.